DTM/Aerial_Conquest/map.json

```diff
@@ -97,13 +97,16 @@
 			"message": "&cYou are not allowed to modify terrain here."
 		},
 		{ "type": "enter", "evaluate": "deny", "teams": ["blue"], "regions": ["red-spawn-protection"], "message": "&cYou may not enter the enemy spawn." },
-		{ "type": "enter", "evaluate": "deny", "teams": ["red"], "regions": ["blue-spawn-protection"], "message": "&cYou may not enter the enemy spawn." }
+		{ "type": "enter", "evaluate": "deny", "teams": ["red"], "regions": ["blue-spawn-protection"], "message": "&cYou may not enter the enemy spawn." },
+		{ "type": "build", "evaluate": "deny", "teams": ["red", "blue"], "regions": ["height-limit"], "message": "&cYou reached the height-limit for this map."}
```

Comment:
You made a filter to prevent players from building in the height limit, but why? There's already a filter up at line 96, why not just add the height-limit region to that? Looking through all these commits, you did this same exact thing for every map JSON you edited.

Reply:
It gives a different message, which tells the player it's a build height instead of a region that the map maker intended for them not to go into I suppose. Either way could have worked, could be something we do for next time?